src/sentry/api/endpoints/organization_measurements_meta.py

```diff
@@ -15,7 +15,7 @@
 @region_silo_endpoint
 class OrganizationMeasurementsMeta(OrganizationEventsEndpointBase):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.EXPERIMENTAL,
```

Comment:
This is meant for custom measurements (like, the metrics kind), that product area is kind of in flux right now, so marking as experimental